src/Compilers/CSharp/Portable/Binder/Binder_Lookup.cs

```diff
@@ -369,6 +370,76 @@ private void LookupMembersInSubmissions(LookupResult result, TypeSymbol submissi
             nonViable.Free();
         }
 
+        protected bool IsUsingAlias(ImmutableDictionary<string, AliasAndUsingDirective> usingAliases, string name, bool callerIsSemanticModel)
```

Comment:
>IsUsingAlias [](start = 23, length = 12)

New methods in this file were moved from Imports.cs with minor adjustments like an addition of extra parameters which are replacing information that used to be in members of Imports.